### PR TITLE
fix(3217431965): Header responsiveness is messed up

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Remove CSS and component handling of responsiveness from `WalletButton` and `Header` components. 

Solves #[3217431965](https://starkware.monday.com/boards/2847524005/pulses/3217431965)

---

### Checklist

- [x] Manually tests of the main Application flows are done and passed.
- [ ] New unit / functional tests have been added (whenever applicable).
- [ ] Docs have been updated (whenever relevant).
- [x] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/316)

### CODE REVIEW

File: src/components/UI/WalletButton/WalletButton.js [hunk 1]
```diff
@@ -27,42 +25,19 @@ WalletButton.propTypes = {
...
-      className={toClasses(styles.accountWalletButton, styles[breakpoint.toLowerCase()])}
+      className={styles.accountWalletButton}
       colorBackground={colorWhiteOp10}
       colorBackgroundHover={colorDarkBlueGray}
       colorBorder={colorOrangeSoda}
       colorText={colorWhite}
-      iconLeft={renderWalletLogo()}
-      iconRight={renderChainLabel()}
-      text={getText()}
+      iconLeft={<DynamicIcon path={logoPath} size={20} />}
```
Comment (yairans-starkware) [inline src/components/UI/WalletButton/WalletButton.js:38]: maybe size 22 it's also ok so u can use `CircleLogoSize.XS`

Comment (yairans-starkware) [reply src/components/UI/WalletButton/WalletButton.js:38]: beside that LGTM

Comment (dan-ziv) [reply src/components/UI/WalletButton/WalletButton.js:38]: I don't want `WalletButton` will be depends on `CircleLogo`. It doesn't make sense

Comment (yairans-starkware) [reply src/components/UI/WalletButton/WalletButton.js:38]: seems like we have to hold `IconSize` enum then

Comment (dan-ziv) [reply src/components/UI/WalletButton/WalletButton.js]: done

**Review by yairans-starkware — DISMISSED** [review]

**Pull request merged**